kubejs/server_scripts/mods/AE2.js

```diff
@@ -460,7 +460,31 @@ ServerEvents.recipes(event => {
         "result": {
             "item": "ae2:printed_silicon"
         }
-    }).id('kubejs:ae2/silicon_print')
+    }).id('ae2:inscriber/silicon_print')
+    const wafers = [
+        ['gtceu:', 'silicon_wafer'],
+        ['gtceu:', 'phosphorous_wafer'],
+        ['gtceu:', 'naquadah_wafer'],
+        ['gtceu:', 'neutronium_wafer'],
+        ['kubejs:', 'universe_wafer']
+    ]
+    wafers.forEach(wafer => {
+        event.custom({
+            "type": "ae2:inscriber",
+            "ingredients": {
+                "middle": {
+                    "item": (wafer[0] + wafer[1])
+                },
+                "top": {
+                    "item": "ae2:silicon_press"
+                }
+            },
+            "mode": "inscribe",
+            "result": {
+                "item": "ae2:printed_silicon"
```

Comment:
I would have guessed that you'd get an increasing number of printed silicon, based on how the different tiers of wafers work in the engraver.

Reply:
Good point. Didn't think to look into seeing if AE2 Inscribers can output more than one item at once, but they can.